src/disturbances/initialize_disturbances.cpp

```diff
@@ -74,15 +74,15 @@ MagneticDisturbance InitMagneticDisturbance(const std::string initialize_file_pa
   return mag_disturbance;
 }
 
-GeoPotential InitGeoPotential(const std::string initialize_file_path) {
+Geopotential InitGeoPotential(const std::string initialize_file_path) {
```

Comment:
```suggestion
Geopotential InitGeopotential(const std::string initialize_file_path) {
```

[IMO] Init関数名側も小文字にした方がよさそうです．

Reply:
修正したのでCI通ったらマージします。